plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,120 @@
+import { Link } from "react-router";
+import prettycron from "prettycron";
+import PropTypes from "prop-types";
+import React from "react";
+import { Tooltip } from "reactjs-components";
+
+import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbSupplementalContent from "#SRC/js/components/BreadcrumbSupplementalContent";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import Icon from "#SRC/js/components/Icon";
+import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
+
+function ItemStatus({ item: { jobRuns } }) {
+  if (!jobRuns) {
+    return null;
+  }
+  const { longestRunningActiveRun } = jobRuns;
+  if (
+    !longestRunningActiveRun ||
+    !longestRunningActiveRun.tasks.longestRunningTask
+  ) {
+    return null;
+  }
+  const { status } = longestRunningActiveRun.tasks.longestRunningTask;
+
+  return (
+    <BreadcrumbSupplementalContent>
+      <div className="service-page-header-status muted">
+        ({TaskStates[status].displayName})
+      </div>
+    </BreadcrumbSupplementalContent>
+  );
+}
+
+function ItemSchedule({ item: { schedules } }) {
+  if (!schedules || !schedules.nodes.length || !schedules.nodes[0].enabled) {
+    return null;
+  }
+  const { cron } = schedules.nodes[0];
+
+  return (
+    <BreadcrumbSupplementalContent>
+      <Tooltip
+        content={prettycron.toString(cron)}
+        maxWidth={250}
+        wrapText={true}
+      >
+        <Icon color="grey" id="repeat" size="mini" />
+      </Tooltip>
+    </BreadcrumbSupplementalContent>
+  );
+}
+
+function getBreadcrumb(item, id = "", name = "") {
+  const isDetailPage = [...item.path, item.id].join(".") === id;
+  const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
+
+  return (
+    <Breadcrumb key={id} title="Jobs">
+      <BreadcrumbTextContent>
+        <Link to={link}>{name === "" ? "Jobs" : name}</Link>
+      </BreadcrumbTextContent>
+      {isDetailPage ? <ItemSchedule item={item} /> : null}
+      {isDetailPage ? <ItemStatus item={item} /> : null}
+    </Breadcrumb>
+  );
+}
+
+function getBreadcrumbList(item) {
+  if (item == null) {
+    return [];
+  }
+
+  const pathSegments = [...item.path, item.name];
```

Comment:
Wonder why here we go with a `name` while at line 56 https://github.com/dcos/dcos-ui/pull/3027/files#diff-2dbad4fb56742926b95061ce75f631b7R56 we go with `id`?

Reply:
argh, that should be name as well in the other place 🙈